README.md

```diff
@@ -1,142 +1,191 @@
+The [specification](docs/specification.md) document contains more details and is used as a basis for the smart contract development.
 
-The [specification](docs/specification.md) document is the specification used as a basis for the smart contract development.
+BaseDAO uses a runtime configuration, which is part of the overall storage of the
+contract, meaning that DAOs with different logic and needs can be obtained by
+simply providing a different initial storage.
 
-Documentation of the actually implemented smart contract is generated automatically.
-* [TrivialDAO](https://github.com/tqtezos/baseDAO/blob/autodoc/master/TrivialDAO.md)
-* [GameDAO](https://github.com/tqtezos/baseDAO/blob/autodoc/master/GameDAO.md)
-* [RegistryDAO](https://github.com/tqtezos/baseDAO/blob/autodoc/master/RegistryDAO.md)
-* [TreasuryDAO](https://github.com/tqtezos/baseDAO/blob/autodoc/master/TreasuryDAO.md)
+Aside from the specification, that contains detail about the configuration as
+well, there are also [example DAOs](#example-daos) below.
 
-## Prerequisites
+## Build instructions
 
-Further steps are given in assumption that `stack` tool is used to compile the project.
-For how to obtain `stack` see [The Haskell Tool Stack](https://docs.haskellstack.org/en/stable/README/) tutorial.
+You will need the [ligo executable](https://ligolang.org/docs/intro/installation) installed to generate the contract.
 
-Also, you may need the very basic knowledge of Haskell and Lorentz if choosing the Lorentz variant. Consider reading [the Lorentz documentation](https://gitlab.com/morley-framework/morley/-/blob/1fdefdb8c081235971cacc002b6704b709349d5c/code/lorentz/README.md).
-
-## Build Instructions
+You can use the [`Makefile`](./Makefile) to build the LIGO contract by simply running:
+```sh
+make out/baseDAO.tz
+```
+which will use `ligo compile-contract` and save the result in `out/baseDAO.tz`.
 
-You can use `stack build` to build `baseDAO` executable.
+You will also need a contract initial storage, which can be obtained with the
+`ligo compile-storage` command, you can also use:
+```sh
+make all
+```
+to generate the contract as well as all the [example DAOs](#example-daos)
+defined below.
 
-For how to obtain `stack` see [The Haskell Tool Stack](https://docs.haskellstack.org/en/stable/README/) tutorial.
+See the [`Makefile`](./Makefile) for details about these and the other commands.
 
-## Usage
+## Originating the contract
 
-Run `stack exec -- baseDAO --help` to see available commands.
+This contract is too large to fit inside the current Tezos origination limit.
 
-### Deploying a contract
+To overcome this limitation we advice to use the [morley-large-originator](https://gitlab.com/morley-framework/morley/-/tree/master/code/morley-large-originator)
+tool or the workaround that it's described there.
 
-Deploying a TZIP-16-compliant baseDAO contract requires the following steps.
+Note: as explained in its README the latest version of its binary can just be
```

Comment:
:thinking: I think there are nightly scheduled tests that will indeed not contain the binary.

I will check that out as it may make more sense to change that in the `morley` side (and remove the `just` from here).